src/styles/blocks/header.scss

```diff
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: $duration;
```

Comment:
You should specify transition properties directly
Default is all, but it could affect old browsers' performance 